contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -56,13 +48,13 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
 
   /// @notice Gets the contracts's configuration
   /// @return config
-  function getConfig() external view returns (Config memory) {
+  function getConfig() external view returns (TermsOfServiceAllowListConfig memory) {
     return s_config;
   }
 
   /// @notice Sets the contracts's configuration
-  /// @param config - See the contents of the TermsOfServiceAllowList.Config struct for more information
-  function updateConfig(Config memory config) public onlyOwner {
+  /// @param config - See the contents of the ITermsOfServiceAllowList Config struct for more information
```

Comment:
This should say "See the contents of the TermsOfServiceAllowListConfig struct in ITermsOfServiceAllowList.sol ..."